batch/scripts/unify_projection.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+# requires arguments
+# -s | --source
+# -T | --target
+#      --target_crs
+
+ME=$(basename "$0")
+. get_arguments.sh "$@"
+
+echo "Reproject to a common CRS"
+
+src_count=0
+CMD_ARGS=()
+
+for s in ${SRC[@]}; do
+  source_dir="SRC_${src_count}"
+  mkdir -p "$source_dir"
+
+  echo "Now recursively downloading $s to $source_dir"
+  if [[ $s == gs://* ]]; then
+    gsutil cp -m -r "$s" "$source_dir"
+  elif [[ $s == s3://* ]]; then
+    aws s3 cp --no-progress "$s" "$source_dir"
+  fi
+  echo "Done downloading $s to $source_dir"
+
+  reprojected_dir="REPROJECTED_${src_count}"
+  mkdir -p "$reprojected_dir"
+
+  cd $source_dir
+  for d in $((tree -dfi)); do
```

Comment:
I think you mean $(tree -dfi) here (only one set of parens).  Also, I guess we are sure the docker has the 'tree' command in it?  You could use 'find' here as well.

I guess this may not matter right now, since our source URIs don't current have sub-directories of files.

Reply:
I wasn't sure, I'll admit that was a try-it-and-see commit. It turns out we don't, and I've replaced the tree command with a find. Thanks for the parenthesis hint!